src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -260,4 +258,11 @@
       </cd-info-card>
     </div>
   </cd-info-group>
+
+  <ng-template #logsLink>
+    <ng-container *ngIf="permissions.log.read">
+      <p class="logs-link"
+         i18n>&rarr; See <a routerLink="/logs">Logs</a> for more details.</p>
```

Comment:
Tip: what about replacing that arrow with some FA icon?

Reply:
Done.